ni-libraries/include/FRC_NetworkCommunication/UsageReporting.h

```diff
@@ -98,6 +98,7 @@ namespace nUsageReporting
         kFramework_Iterative = 1,
         kFramework_Simple = 2,
         kFramework_CommandControl = 3,
+        kFramework_Periodic = 4,
```

Comment:
This will get overwritten the next time we import NI libraries... but that's a different issue.